<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QualityManageDetail</class>
 <widget class="QWidget" name="QualityManageDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1696</width>
    <height>752</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1696</width>
    <height>752</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1696</width>
    <height>752</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>1670</width>
     <height>80</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1670</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1670</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget
{
background: #37ADBA;
border-radius: 10px 10px 0px 0px;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>467</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: PingFang SC;
font-weight: bold;
font-size: 26px;
color: #FFFFFF;
line-height: 42px;</string>
      </property>
      <property name="text">
       <string>质控品编辑</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>466</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>88</y>
     <width>1670</width>
     <height>150</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1670</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1670</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_2
{
background: #FFFFFF;
border-radius: 10px;
}</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>26</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #333333;
line-height: 42px;</string>
    </property>
    <property name="text">
     <string>质控名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_periodOfValidity">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>380</width>
      <height>58</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>58</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>58</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);
font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>90</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #333333;
line-height: 42px;</string>
    </property>
    <property name="text">
     <string>批号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_batchNumber">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>73</y>
      <width>380</width>
      <height>58</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>58</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>58</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);
font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_name">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>9</y>
      <width>380</width>
      <height>58</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>58</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>58</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);
font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>26</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #333333;
line-height: 42px;</string>
    </property>
    <property name="text">
     <string>有效期</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_table" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>1165</width>
     <height>502</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1165</width>
     <height>502</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1165</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_table
{
background: #FFFFFF;
border-radius: 10px;
}
QTableWidget#tableWidget
{
background: #FFFFFF;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
border-radius: 10px;
color: #333333;
line-height: 42px;
}
QTableWidget::item {
    text-align: center;
}
QHeaderView {
    font-family: PingFang SC;
	font-weight: 500;
	font-size: 22px;
	color: #6D7E92;
	line-height: 42px;
}

QHeaderView::section {
    background-color: #EFF0F2;
	border-top: 1px solid #E1EBF6;
	border-left: none;
	border-bottom: 1px solid #E1EBF6;
	border-right: none;
	line-height: 42px;
}
QTableWidget::item:selected {
    background: #D4F2F5;
	border-top: none;
	border-left: none;
	border-bottom: 1px solid #22A4B3;
	border-right: none;
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>1173</x>
     <y>240</y>
     <width>506</width>
     <height>502</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>506</width>
     <height>502</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>506</width>
     <height>502</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_4
{
background: #FFFFFF;
border-radius: 10px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>227215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #666666;
line-height: 42px;</string>
        </property>
        <property name="text">
         <string>项目</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_project">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #F2F7FB;
border-radius: 25px;
border: 1px solid rgba(170,189,212,0.6);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>82</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>82</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #666666;
line-height: 42px;</string>
        </property>
        <property name="text">
         <string>靶值范围</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_valTargetStart">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>89</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #F2F7FB;
border-radius: 25px;
border: 1px solid rgba(170,189,212,0.6);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_valTargetEnd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #F2F7FB;
border-radius: 25px;
border: 1px solid rgba(170,189,212,0.6);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #666666;
line-height: 42px;</string>
        </property>
        <property name="text">
         <string>目标值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_valTarget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #F2F7FB;
border-radius: 25px;
border: 1px solid rgba(170,189,212,0.6);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #666666;
line-height: 42px;</string>
        </property>
        <property name="text">
         <string>SD值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_valSD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #F2F7FB;
border-radius: 25px;
border: 1px solid rgba(170,189,212,0.6);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #666666;
line-height: 42px;</string>
        </property>
        <property name="text">
         <string>单位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_unit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #F2F7FB;
border-radius: 25px;
border: 1px solid rgba(170,189,212,0.6);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
